lib/components/__tests__/clock.spec.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+import { Clock } from '../clock/clock';
+
+describe('clock', () => {
+    it('should render', () => {
```

Comment:
I will leave this up to your judgment, I am not really focusing on component testing and I am sure this is the only component I tested throughout the whole time.